decidim-core/lib/decidim/core/test/shared_examples/comments_examples.rb

```diff
@@ -1039,9 +1059,37 @@
 
       it "shows a message indicating that comments are disabled" do
         visit resource_path
-        expect(page).to have_content("Comments are disabled at this time")
+        expect(page).to have_content("Comments are currently disabled, only administrators can reply or post new ones.")
         expect(page).to have_no_content("You need to be verified to comment at this moment")
       end
+
+      context "when the user is an administrator" do
+        let!(:user) { create(:user, :admin, :confirmed, organization:) }
+
+        it_behaves_like "can answer comments"
+      end
+
+      context "when the user has a role of user manager" do
+        let!(:user) { create(:user, :user_manager, :confirmed, organization:) }
+
+        it_behaves_like "can answer comments"
+      end
+
+      context "when the user has an evaluator role in the same participatory space" do
+        let!(:valuator_role) { create(:participatory_process_user_role, role: :valuator, user:, participatory_process: participatory_space) }
+
+        it_behaves_like "can answer comments"
+      end
+
+      context "when the user has an evaluator role in a different participatory space" do
```

Comment:
Could we also add some specs for other space types?

Reply:
done